airflow/example_dags/example_xcomargs.py

```diff
@@ -66,4 +66,4 @@ def print_value(value):
     xcom_args_a = print_value("first!")  # type: ignore
     xcom_args_b = print_value("second!")  # type: ignore
 
-    bash_op1 >> xcom_args_a >> xcom_args_b >> bash_op2
+    bash_op1 >> xcom_args_a >> xcom_args_b >> bash_op2  # type: ignore
```

Comment:
oh ! is MyPy complaining here?

Reply:
I don't get it and I would prefer to solve it in other PR as I belived this will required either myplugin or some type changes in `@task` decorator. The original error:
```
airflow/example_dags/example_xcomargs.py:66: error: Value of type variable "T"
of "print_value" cannot be "str"
        xcom_args_a = print_value("first!")
                      ^
airflow/example_dags/example_xcomargs.py:67: error: Value of type variable "T"
of "print_value" cannot be "str"
        xcom_args_b = print_value("second!")
                      ^
airflow/example_dags/example_xcomargs.py:69: error: Unsupported left operand
type for >> ("BashOperator")
        bash_op1 >> xcom_args_a >> xcom_args_b >> bash_op2
```

Reply:
ok, fixed? Use `@task()` instead of `@task`

Reply:
hmm interesting, any idea here @casassg -- "@task" should work.

This can be addressed in a separate PR though

Reply:
Both `@task` and `@task()` works, however the type annotation of this decorator seems to be confusing mypy (althought it looks valid to me)